descriptors/annotators/io/LoadPointCloud.xml

```diff
@@ -0,0 +1,62 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<taeDescription xmlns="http://uima.apache.org/resourceSpecifier">
+  <frameworkImplementation>org.apache.uima.cpp</frameworkImplementation>
+  <primitive>true</primitive>
+  <annotatorImplementationName>rs_loadPointCloud</annotatorImplementationName>
+  <analysisEngineMetaData>
+    <name>LoadPointCloud</name>
+    <description>Load Point clouds from files.</description>
+    <version>1.0</version>
+    <vendor/>
+    <configurationParameters>
+        <configurationParameter>
+            <name>folder</name>
+            <type>String</type>
+            <multiValued>false</multiValued>
+            <mandatory>true</mandatory>
+        </configurationParameter>
+        <configurationParameter>
+            <name>filenames</name>
+            <type>String</type>
+            <multiValued>true</multiValued>
+            <mandatory>true</mandatory>
+        </configurationParameter>
+    </configurationParameters>
+    <configurationParameterSettings>
+        <nameValuePair>
+            <name>folder</name>
+            <value>
+                <string>/home/anindex/</string>
```

Comment:
In the latest commit, I use the simplest solution std::string.find to find ".pcd". Regex could be a better solution in some case. 
We need to define some "scenarios". e.g "Kinect" that loads depth, rgb and cloud. "Cloud" just loads clouds, etc. We also combine with the naming template ofc.